plugins/grafana-sankey-plugin/README.md

```diff
@@ -0,0 +1,44 @@
+# Grafana Panel Plugin Template
+
+[![Build](https://github.com/grafana/grafana-starter-panel/workflows/CI/badge.svg)](https://github.com/grafana/grafana-starter-panel/actions?query=workflow%3A%22CI%22)
+
+This template is a starting point for building Grafana Panel Plugins in Grafana 7.0+
+
+## What is Grafana Panel Plugin?
+
+Panels are the building blocks of Grafana. They allow you to visualize data in different ways. While Grafana has several types of panels already built-in, you can also build your own panel, to add support for other visualizations.
+
+For more information about panels, refer to the documentation on [Panels](https://grafana.com/docs/grafana/latest/features/panels/panels/)
+
+## Getting started
+
+1. Install dependencies
+
+   ```bash
+   yarn install
+   ```
+
+2. Build plugin in development mode or run in watch mode
+
+   ```bash
+   yarn dev
+   ```
+
+   or
+
+   ```bash
+   yarn watch
+   ```
+
+3. Build plugin in production mode
+
+   ```bash
+   yarn build
+   ```
+
+## Learn more
+
+- [Build a panel plugin tutorial](https://grafana.com/tutorials/build-a-panel-plugin)
```

Comment:
From what I gather this plugin is based on simplePanel plugin, but it's specific for dsiplaying a Sankey Diagram, so maybe keeping original links and Readme might not be accurate.

Are you preserving these files for proper copyright attribution?

Reply:
I kept this README file to help developers understand how to build and test plugin. Yes you are right, I changed some wording to make it more proper to our case.

Reply:
Thanks!

Reply:
I added some contents specific to our sankey-panel-plugin.